packages/database/prisma/schema.prisma

```diff
@@ -154,6 +155,14 @@ model MerchantProduct {
   merchantPrices    MerchantPrice[]
 }
 
+model MerchantSession {
+  id                String          @id @default(uuid())
+  identifier        String
+  merchantAccountId String
+  merchantAccount   MerchantAccount @relation(fields: [merchantAccountId], references: [id], onDelete: Cascade)
+  purchase          Purchase[]
+}
```

Comment:
This is called `MerchantSession` now and it has an `identifier` which in the case of Stripe merchant purchases will be a `checkoutSessionId`.